package.json

```diff
@@ -35,14 +38,14 @@
   "devDependencies": {
     "@defi-wonderland/crypto-husky-checks": "1.3.0",
     "@testing-library/jest-dom": "6.1.2",
-    "@testing-library/react": "14.0.0",
+    "@testing-library/react": "^14.2.1",
```

Comment:
remove carets, to lock the dependency version

Reply:
https://github.com/defi-wonderland/web3-react-boilerplate/commit/af82dd526ba9023b2a88ca17bbe35c75120fa320